### PR TITLE
feat: styles setup

### PR DESCRIPTION
**Pull request by simboonlong**
for a start

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,4 +1,14 @@
+import styled from "@emotion/styled";
 import React from "react";
 import { Router } from "./Router";
+import { base } from "./styles/base";
 
-export const App: React.FunctionComponent = () => <Router />;
+export const App: React.FunctionComponent = styled(({ className }) => {
+  return (
+    <div className={`${className} text-gray-800`}>
```
Comment (yehjxraymond) [inline src/App.tsx:8]: Thinking if it make sense to put all the styling here. I'm thinking if a component `BaseStyle` and you can put both the class and the base() inside... wdyt?

File: src/styles/base/index.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { global } from "./global";
+import { typography } from "./typography";
+
+export const base = (): string => {
+  return `
+    ${global()}
```
Comment (yehjxraymond) [inline src/styles/base/index.tsx:6]: I think tailwind allows you to customise the base layer, not sure if this method of overwriting is better... I'm guessing the other alternative is to install the font and use the font family as default on tailwind config. wdyt?

Discussion: https://github.com/tailwindcss/discuss/issues/277

Comment (simboonlong) [reply src/styles/base/index.tsx]: i've set tailwind config as the base and source of truth, there is some workaround though, need to put the config within ./src folder... abit erxin

File: src/App.tsx (hunk 1 above)
Comment (simboonlong) [reply src/App.tsx]: either way also can, for now maybe lets try embracing tailwind way of setup then we see how it goes

**Pull request merged**